src/jesse_schema_validator.hrl

```diff
@@ -49,11 +52,13 @@
 -define(MINLENGTH,            <<"minLength">>).
 -define(MAXLENGTH,            <<"maxLength">>).
 -define(ENUM,                 <<"enum">>).
+-define(CONST,                 <<"const">>).
 -define(FORMAT,               <<"format">>).               % NOT IMPLEMENTED YET
 -define(DIVISIBLEBY,          <<"divisibleBy">>).
 -define(DISALLOW,             <<"disallow">>).
 -define(EXTENDS,              <<"extends">>).
 -define(ID,                   <<"id">>).
+-define(IDv6,                 <<"$id">>).
```

Comment:
since `$id` is the future, and there are not many occurrences of `?ID`, I suggest we rename the old `?ID` to... `?ID_OLD` or a better suggestion, and make `?ID` point to `$id`

Reply:
Fixed